test/integration/default/files/_mapdata/opensuse-tumbleweed.yaml

```diff
@@ -45,16 +45,14 @@ values:
     containers:
       base: /opt/containers
     required_pkgs:
-      - git
-      - python3-pip
+      - python3-podman
+      - python3-toml
     salt_compat:
-      pips:
-        - docker
-      pkgs:
-        - python3-pip
```

Comment:
Note that this file is currently unused (but templated), there are no tests yet sadly. Still:

question: Do we need to install the `docker` library for `docker` module compatibility on SUSE? Because this change is not applied to the parameters file above, which would still install `python3-pip` and `docker` via pip.

Reply:
True, I don't use this with Docker hence I did not notice. I added `python3-docker` now, which is the correct package for the Docker Python library, but have not tested if any other changes are required.